### DISCUSSION TITLE
why is directus graphql endpoint `/graphql/system` ?

### DISCUSSION THREAD

**Discussion by anlexN** [directus/directus#15626] [Archive]
why does **directus** design that way?

[graphql.org](https://graphql.org/learn/serving-over-http/) and [github](https://docs.github.com/en/graphql/guides/forming-calls-with-graphql#the-graphql-endpoint) all use `/graphql` as endpoint.

**Comment by azrikahar** [comment]
The general graphql endpoint is _still_ `/graphql`! 

`/graphql/system` is only used for system collections, not collections that you have created yourself. 

It is to prevent naming conflicts (such as when someone creates their own `users` collection, which is different than the system `users` collection). You can read more about this design choice over here: https://docs.directus.io/reference/introduction.html#system-data-in-graphql

**Reply by anlexN** [reply]
thank you very much
